core/src/main/java/tech/pegasys/web3signer/core/multikey/metadata/parser/SigningMetadataModule.java

```diff
@@ -81,4 +84,13 @@ public void serialize(
       gen.writeString(value.toString());
     }
   }
+
+  public static class NioPathSerializer extends JsonSerializer<Path> {
```

Comment:
Name seems a bit odd. But maybe irrelevant if we can use the ToStringSerializer that siladu suggested.

Reply:
👍🏼